src/js/big-play-button.js

```diff
@@ -34,7 +34,22 @@ class BigPlayButton extends Button {
    * @listens click
    */
   handleClick(event) {
-    this.player_.play();
+    const playPromise = this.player_.play();
+
+    const cb = this.player_.getChild('controlBar');
+    const playToggle = cb && cb.getChild('playToggle');
+
+    if (!playToggle) {
+      return;
```

Comment:
Strictly, focus ought to move to something which is actionable, but it can move to something which isn't if pressing Tab from there will get the user to an actionable control (rather than, for example, the next focusable element _after_ the video.js player)